### ISSUE TITLE
access to signals on the can bus

### ISSUE THREAD

**Issue by sgnes** [eerimoq/cantools#120]
Hello,

I'm looking for a tool to access to the signals on the can bus, like get_signal_value_phys(message name, signal name), and after check the document for cantools, I don't find this kind of interface.

Would you please help check this point?
If we don't have ,shall we have a class like "Device" to hold the signals values, and user can get it easily?
Thanks.

**Comment by eerimoq** [comment]
You're right, there is no such API. Feel free to design it and submit a PR.

**Comment by sgnes** [comment]
@eerimoq Thanks for your reply, I will try to create a PR soon.
